File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.java
import java.util.*;

class Solution {
    public boolean solution(String s) {
        boolean answer = s.length() == 4 || s.length() == 6;
        int bowl = 0;
        for(int i = 0; i < s.length(); i++){
            try{
                bowl = Integer.parseInt(s.substring(i, i+1));
            }catch(Exception e){
                return false;
            }
        }
        return answer;
    }
}

File: 프로그래머스/lv1/12918. 문자열 다루기 기본/문자열 다루기 기본.js
function solution(s) {
    var answer = s.length == 4 || s.length == 6;
    for(let i = 0; i < s.length; i++){
        if(isNaN(parseInt(s.substr(i, 1)))){
            return false;
        }
    }
    return answer;
}